docs/API.md

```diff
@@ -190,7 +190,7 @@ Octal permission specifying the mode the directory should be created with: e.g.
 
 ### gulp.task([name,] fn)
 
-Define a task using [Undertaker].
+Define a task exposed to Gulp command line; inherited from [Undertaker].
```

Comment:
I changed it in https://github.com/gulpjs/gulp/commit/9875ec578cc7bda9947cefa0c2c5fd5c29d7696f